units/provisioning_setup_host.go

```diff
@@ -823,13 +824,21 @@ func mountLinuxVolume(ctx context.Context, env evergreen.Environment, h *host.Ho
 
 	// write to /etc/fstab so the volume is mounted on restart
 	// use the UUID which is constant over the life of the filesystem
```

Comment:
Move this comment down to the place where you run `tee`.

Reply:
done